### PR TITLE
copied block template for gtranslate

### PR DESCRIPTION
**Pull request by ghost**
# Pull Request

Closes Issue #X

### CODE REVIEW

**Comment by ghost** [comment]
https://github.com/UTCWeb/utccloud/issues/1471

File: apps/drupal-default/particle_theme/templates/block/block--gtranslate.html.twig [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    'pb-0',
+    'flex',
+    'flex-row-reverse',
+    'block',
+    'block-' ~ configuration.provider|clean_class,
+    'block-' ~ plugin_id|clean_class,
+  ]
+%}
+
+<style>.gtranslate > select {
```
Comment (ghost) [inline apps/drupal-default/particle_theme/templates/block/block--gtranslate.html.twig:42]: If I need how to learn how to do this properly, let me know.

Comment (ghost) [reply apps/drupal-default/particle_theme/templates/block/block--gtranslate.html.twig:42]: These rules give it a similar look to how it was done in OUC, minus the Google branding.

Comment (bmartinez287) [reply apps/drupal-default/particle_theme/templates/block/block--gtranslate.html.twig:42]: Looks good to me. There's different ways of approaching it and it all depends on how much data, variables the twig files deal with. But for this specific scenario that looks fine to me.
So for something as large as a card I would not recommend it but this is very atomic.

**Review by bmartinez287 — APPROVED** [review]

**Review by MegShad — APPROVED** [review]

**Review by UTCGilligan — APPROVED** [review]

**Pull request merged**